### PR TITLE
feat: add some description for secure multiplayer project

### LINKED ISSUE #44192: Add instructions to real time multiplayer game
Edit from @moT01: See [comment below](https://github.com/freeCodeCamp/freeCodeCamp/issues/44192#issuecomment-1190403770) for the new focus of this issue.

Related to #39721

The Secure Real Time Multiplayer game is missing user stories
reported here: https://forum.freecodecamp.org/t/where-are-the-user-stories/485714

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #44192

This is minimum effort, but it adds some instructions.

The alternative would be to bring a lot of what's in the tests in the instructions and make the tests more terse

### CODE REVIEW

**Review by Sboonny — DISMISSED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

File: curriculum/challenges/english/09-information-security/information-security-projects/secure-real-time-multiplayer-game.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ When you are done, make sure a working demo of your project is hosted somewhere
...
+Create a secure multiplayer game in which each player can move their avatar, there is at least a collectible item, and the rank of the players is calculated based on their score.
+
+For details consult the tests below.
+
+Make sure that your game is secure! Include these security measures:
+
+- the client should not be able to guess/sniff the MIME type
+- prevent XSS attacks
+- do not cache anything from the website in the client
+- the headers say that the site is powered by "PHP 7.4.3"
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/09-information-security/information-security-projects/secure-real-time-multiplayer-game.md:30]:
```suggestion
Create a secure multiplayer game in which each player can move their avatar, there is at least one collectible item, and the rank of the players is calculated based on their score.

For details consult the tests below.

Make sure that your game is secure! Include these security measures:

- The client should not be able to guess/sniff the MIME type
- Prevent XSS attacks
- Do not cache anything from the website in the client
- The headers say that the site is powered by `PHP 7.4.3`
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Minor request

File: curriculum/challenges/english/09-information-security/information-security-projects/secure-real-time-multiplayer-game.md (hunk 1 above)
Comment (ieahleen) [reply curriculum/challenges/english/09-information-security/information-security-projects/secure-real-time-multiplayer-game.md:30]: minor request granted

**Review by Sboonny — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Pull request merged**